doc/api/crypto.md

```diff
@@ -1883,6 +1886,23 @@ the corresponding digest algorithm. This does not work for all signature
 algorithms, such as `'ecdsa-with-SHA256'`, so it is best to always use digest
 algorithm names.
 
+### crypto.isKeyObject(value)
```

Comment:
The order of APIs appears to be incorrect, assuming we are still sorting alphabetically.